user/managing-os/hvm.md

```diff
@@ -152,7 +181,8 @@ drive             : None
 timezone          : localtime
 ~~~
 
-Note how the MAC addresses differ between those two otherwise identical VMs. The IP addresses assigned by Qubes will also be different of course to allow networking to function properly:
+Note how the MAC addresses differ between those two otherwise identical VMs.
```

Comment:
Since R4.0, all qubes have the same mac address by default. It makes keeping windows license activation easier for template-based vms.

Reply:
Looks like the entire preceding example output and the text before it needs to be updated in order to fix this.

If you don't want to do that in this PR, you don't have to, @unman. Let me know what you decide.

Reply:
@andrewdavidwong Please merge this, and I'll look separately at the MAC address treatment.